src/Plugin/monitoring/SensorPlugin/TidetimesSensorPlugin.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Drupal\tide_core\Plugin\monitoring\SensorPlugin;
+
+use Drupal\monitoring\Result\SensorResultInterface;
+use Drupal\monitoring\SensorPlugin\ExtendedInfoSensorPluginInterface;
+use Drupal\monitoring\SensorPlugin\SensorPluginBase;
+
+/**
+ * Monitors the Section API connection.
+ *
+ * @SensorPlugin(
+ *   id = "tide_times_sensor",
+ *   label = @Translation("Tide Times Sensor"),
+ *   description = @Translation("Monitors tide modules."),
+ * )
+ */
+class TidetimesSensorPlugin extends SensorPluginBase implements ExtendedInfoSensorPluginInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function resultVerbose(SensorResultInterface $result) {
+    return [
+      '#type' => 'inline_template',
+      '#template' => '
+      <div class="system-info-collapsible">
+        {% for section, data in sections %}
```

Comment:
Provide a more robust fallback mechanism in case some data is missing: 
```suggestion
        {% for section, data in sections ?? {} %}
```